### ISSUE TITLE
Add Infusate, Tracer, and TracerLabel name fields to the PeakGroup format class as searchable

### ISSUE THREAD

**Issue by hepcat72** [Princeton-LSI-ResearchComputing/tracebase#555]
### Inspiration

I wanted to search the infusate name to see if the parens are actively updating to square brackets by searching for Tracer contains "[" and realized I couldn't do it.

### Description

Add Infusate, Tracer, and TracerLabel name fields to the peak groups format.  They're already in the PeakData format and tracer is in the FCirc format.  PeakGroup was just overlooked.

### Alternatives

none

### Dependencies

none

### Comment

none

-----

### Assumptions

- List of assumptions that the code will not explicitly address/check
- E.g. We will assume input is correct (explaining why there is no validation)

### Requirements

- [ ] 1. List of numbered conditions to be met for the feature
- [ ] 2. E.g. Every column/row must display a value, i.e. cannot be empty
- [ ] 3. Numbers for reference & checkboxes for progress tracking

### Limitations

- A list of things this work will specifically not do
- E.g. This feature will only handle the most frequent use case X

### Affected Components

A tentative list of anticipated repository items that will be changed, labeled
with "add", "delete", or "change".  One item per line.  (Mostly, this will be a
list of files.)

- change: File path or DB table ...
- add: Environment variable or server setting
- delete: External executable or cron job

#### Interface Change description

Describe changes to usage. *E.g. GUI/command-line changes*

#### Code Change Description

Describe code changes planned for the feature. *(Pseudocode encouraged)*

#### Tests

- [ ] 1. A description of at least one test for each requirement above.
- [ ] 2. E.g. Test for req 2 that there's an exception when display value is ''
- [ ] 3. Numbers for reference & checkboxes for progress tracking

**Comment by lparsons** [comment]
I'm not sure which fields would be added by this issue, but I'd like to ensure that "Tracer Compound" gets added. It was there in the previous iterations and makes filtering the data on tracer compound simpler if we don't have to parse an Infusate name, but can match on a column instead.

**Comment by hepcat72** [comment]
There is a PR for this issue (#562). Tracer Compound was not removed. It's still there. I only tweaked the name so that all of the compound fields would group together in the select list. But perhaps we're not on the same page?

**Comment by lparsons** [comment]
Hmmm. I didn't see it in the download data (which is all I'm really looking at right now). I'll have to see if it's some.other more obscure name? Or perhaps something is odd about the column and it didn't parse easily. I'll report back.

**Comment by hepcat72** [comment]
Ah, I see. Yes, we're on different pages.  This issue is a tad ambiguously described, but the inspiration makes it a little clearer.  I created this issue to add those fields to the field select list, because I thought it would be useful to add the entire tracer name to the search fields.  The issue for adding the columns is #541.  Fan's doing that for the list/detail pages and I had started to make the updates to the advanced search formats the other day until I asked for that clarification.

If tracer compound was removed from the formats, I don't recall.  When I said it was still there, I was referring to the search options in the field select list.  Maybe I removed the tracer compound column some time back because I thought it was redundant when I did the multi-tracer update?  If that's the case, and you want it back, then we should create an issue.
